modules/auth/assembly-enabling-authentication.adoc

```diff
@@ -0,0 +1,35 @@
+[id='enabling-authentication']
+= Enabling authentication in {product}
+
+Authentication within {product} facilitates user sign-in, identification, and access to external resources.
+It supports multiple authentication providers.
+
+Usually, set up:
+
+- One provider for sign-in and identification.
+- Additional providers for accessing external resources.
+
+The {product} supports the following authentication providers:
+
+Auth0:: `auth0`
+Atlassian:: `atlassian`
+Microsoft Azure:: `microsoft`
+Microsoft Azure Easy Auth:: `azure-easyauth`
+Bitbucket:: `bitbucket`
+Bitbucket Server:: `bitbucketServer`
+Cloudflare Access:: `cfaccess`
+GitHub:: `github`
+GitLab:: `gitlab`
+Google:: `google`
+Google IAP:: `gcp-iap`
+OIDC:: `oidc`
+Okta:: `okta`
+OAuth 2 Custom Proxy:: `oauth2Proxy`
+OneLogin:: `onelogin`
+SAML:: `saml`
+
+For each provider, follow the dedicated procedure to:
+
+. Set up in your provider the secrets that {product} requires to communicate with the provider.
```

Comment:
```suggestion
. Set up the secrets that {product} requires to communicate with the provider.
```

Reply:
I believe here it is important to distinguish what you are doing on the provider side from what you are doing on the product side. With your suggestion we would loose that.

Or maybe:

```suggestion
. Set up the secrets that {product-short} requires to communicate with the provider, on the provider side, then on {product-short} side.
```

and remove the next line. 

WDYT @linfraze?

Reply:
I do agree. I think it's good practice to start by orienting the user, so my original change was _In the provider settings, set up the secrets..._ but it sounded a little awkward with the lead-in sentence, so I just removed it and figured the details would be shared the associated procedures. Maybe we can say _...in both the provider and {product-short} settings_ ? If that is what we actually mean. Saying _on the [provider/RHDH] side_ is a little informal and vague.